roadservice/core/models.py

```diff
@@ -181,7 +183,10 @@ def has_expert(self):
         return hasattr(self, 'expert')
 
     def notify_expert(self):
-        pass  # TODO
+        sms_sender = SMSSender.get_instance()
```

Comment:
It is best to do these operations here, or in `CountyExpert`'s own `notify` method? (The latter is how the sequence diagram is drawn)